action/dependencies.py

```diff
@@ -173,3 +172,18 @@ def add_repos(repos: str):
                 "/home",
             )
         )
+
+
+def add_packages(arch: str, packages: str) -> Dict[str, str]:
+
+    downloaded_default_packages, downloaded_packages = get_packages(arch, packages)
+
+    if downloaded_packages == []:
+        return {}
+
+    return {
+        "python": {
+            "PYTHON_DEF_PACKAGES": " ".join([f'/var/packages/{package}' for package in downloaded_default_packages]),
+            "PYTHON_PACKAGES": " ".join([f'/var/packages/{package}' for package in downloaded_packages]),
+        }
+    }
```

Comment:
1. So if there are no user-specified packages, the default packages won't get installed as well? I get that it's just `wheel` currently, but that could change, which could lead to non-obvious errors.
2. Why do we need `wheel`? Is `pip` not sufficient?
3. Does this approach work if no packages are specified? Or does `pip` error out?
4. I think we should rename `PYTHON_DEF_PACKAGES` to `PYTHON_DEFAULT_PACKAGES` to make it clearer.

Reply:
Ok, makes sense. I like `INSTALLATION_DEPENDENCIES`. Or `INSTALL_DEPS` if you want it shorter.